sites/default/statement.inc.php

```diff
@@ -21,13 +22,384 @@
+    create__HTML_statement($stmt);
+  } else {
+    create_statement($stmt);
+  }
+}
+/**
+ * This prints a header for documents.  Keeps the brand uniform...
+ *  @param string $pid patient_id
+ *  @param string $direction, options "web" or anything else.  Web provides apache-friendly url links.
+ *  @return outputs to be displayed however requested
+ */
+function report_header_2($stmt,$direction='',$providerID='1') {
+  $titleres = getPatientData($stmt['pid'], "fname,lname,DOB");
+  if ($_SESSION['pc_facility']) {
+    $sql = "select * from facility where id=?";
+    $facility = sqlQuery($sql,array($_SESSION['pc_facility']));
+  } else {
+    $sql = "SELECT * FROM facility ORDER BY billing_location DESC LIMIT 1";
+    $facility = sqlQuery($sql);
+  }
+  $DOB = oeFormatShortDate($titleres['DOB']);
+  /******************************************************************/
+  ob_start();
+  // Use logo if it exists as 'practice_logo.gif' in the site dir
+  // old code used the global custom dir which is no longer a valid
+  ?>
+  <table style="width:7in;">
+    <tr>
+      <td style='width:100px;text-align:top;'>
+        <?php
+        if ($direction == "web") {
+          global $OE_SITE_DIR;
+          $practice_logo = $GLOBALS['webroot']."/sites/default/images/practice_logo.gif";
+          if (file_exists($OE_SITE_DIR."/images/practice_logo.gif")) {
+            echo "<img src='$practice_logo' align='left' style='width:150px;margin:0px;'><br />\n";
+          }
+        } else { //defaults to this
+          global $OE_SITE_DIR;
+          $practice_logo = "$OE_SITE_DIR/images/practice_logo.gif";
+          if (file_exists($practice_logo)) {
+            echo "<img src='$practice_logo' align='left' style='width:125px;margin:0px;'><br />\n";
+          }
+        }
+        ?>
+      </td>
+      <td style='width:40%;'>
+        <em style="font-weight:bold;font-size:1.4em;"><?php echo text($facility['name']); ?></em><br />
+        <?php echo text($facility['street']); ?><br />
+        <?php echo text($facility['city']); ?>, <?php echo text($facility['state']); ?> <?php echo text($facility['postal_code']); ?><br />
+        <?php echo xlt('Phone').': ' .text($facility['phone']); ?><br />
+        <?php echo xlt('Fax').': ' .text($facility['fax']); ?><br />
+        <br clear='all' />
+      </td>
+      <td>
+        <em style="font-weight:bold;font-size:1.4em;"><?php echo text($titleres['fname']) . " " . text($titleres['lname']); ?></em><br />
+        <b style="font-weight:bold;"><?php echo xlt('Chart Number'); ?>:</b> <?php echo text($stmt['pid']); ?><br />
+        <b style="font-weight:bold;"><?php echo xlt('Generated on'); ?>:</b> <?php echo oeFormatShortDate(); ?><br />
+        <b><?php echo xlt('Provider') . ':</b>  '; ?><?php echo getProviderName($providerID); ?> <br />
+      </td>
+    </tr>
+  </table>
+  <?php
+  $output = ob_get_contents();
+  ob_end_clean();
+  return $output;
+}
+
+function create_HTML_statement($stmt) {
```

Comment:
The original is plain text. The other is actually HTML with images from the practice, the CC image, more professional fonts and layout.  The next commit looks a little different again (should fit into a number 9 envelope).  The difference to me is the original was made for a dot matrix printer, this one is made for a laser printer.